packages/atomic/src/components/result-template-components/atomic-result-text/atomic-result-text.tsx

```diff
@@ -51,6 +56,19 @@ export class AtomicResultText {
     }
   }
 
+  private get resultValue() {
+    const value = ResultTemplatesHelpers.getResultProperty(
+      this.result,
+      this.field
+    ) as string;
+
+    if (!value || value.trim() === '') {
```

Comment:
Similar to the other PR, it would be more type safe to check if `value` is a string rather than cast it.

```
if (typeof value !== 'string' || value.trim() === '') {}
```